.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,48 @@
+name: Build and Publish to NPM
+on:
+    push:
+        branches:
+          - master
+          - next
+          - next-major
+          - alpha
+          - beta
+          - development
+
+permissions:
+    contents: read # for checkout
```

Comment:
@adrienne-deriv, can this be at the job level?

Reply:
The job already has `contents: write` permission, so this is redundant, I'll remove it